core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorMetadata.java

```diff
@@ -610,6 +617,9 @@ default void dropView(ConnectorSession session, SchemaTableName viewName)
 
     /**
      * Gets the view data for the specified view name.
```

Comment:
Could we also clarify here that it should return `empty` when object is table or MV?

Ditto below

Reply:
Done